spec/services/invoices/payments/stripe_service_spec.rb

```diff
@@ -138,6 +146,16 @@
       let(:stripe_customer) { create(:stripe_customer, customer:) }
 
       before do
+        allow(Stripe::Customer).to receive(:retrieve)
+          .and_return(Stripe::StripeObject.construct_from(
+            {
+              invoice_settings: {
+                default_payment_method: nil
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.